experimental/dds/tree2/src/shared-tree/sharedTreeView.ts

```diff
@@ -396,8 +361,13 @@ export class SharedTreeView implements ISharedTreeBranchView {
 				}
 			}
 		});
-		branch.on("revertible", (type) => {
-			this.events.emit("revertible", type);
+		branch.on("revertible", (revertible) => {
+			// if there are no listeners, discard the revertible to avoid memory leaks
+			if (!this.events.hasListeners("revertible")) {
```

Comment:
Not necessary for now, but what would it take to prevent the revertible creation in the first place?